### PR TITLE
Important contacts refresh

### PR DESCRIPTION
**Pull request by drewvolz**
This PR introduces a refresh to the important contacts view.

**Main:**
* Simplified list/detail views
* Group contacts by category

**Misc:**
* Whitespace/character changes to YAML
* Filename changes

~ | List | Detail
--|--|--
iOS | ios-list | ios-detail
Android | android-list | android-detail

### CODE REVIEW

**Comment by drewvolz** [comment]
~What am I missing... `npm run validate-data -- --quiet` shows that "category" is an additional prop.~

Ah. Forgot to update the schema.

File: source/views/contacts/contact-list.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      contact,
+    })
+  }
+
+  renderSeparator = () => <ListSeparator />
+
+  renderSectionHeader = ({section: {title}}: any) =>
+    <ListSectionHeader title={title} spacing={{left: 10}} />
+
+  renderFooter = () =>
```
Comment (hawkrives) [inline source/views/contacts/contact-list.js]: Please extract this into a constant outside of the class - it doesn't need to be called as a function. 

Just assign the ListFooter to a constant called "footer" or something. It should be in this file, as it's similar in intent to the styles object.

File: source/views/contacts/contact-list.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import {data} from './data'
+import groupBy from 'lodash/groupBy'
+import toPairs from 'lodash/toPairs'
+import * as c from '../components/colors'
+import type {ContactType} from './types'
+
+const AAO_URL = 'https://github.com/StoDevX/AAO-React-Native/issues/new'
+
+const groupContacts = (contacts: ContactType[]) => {
+  const grouped = groupBy(contacts, b => b.category || 'Other')
```
Comment (hawkrives) [inline source/views/contacts/contact-list.js]: We don't need the `||` here, as we're guaranteed through the schema that our objects will have a category.

File: source/views/contacts/contact-list.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/**
+ * @flow
+ * All About Olaf
+ * iOS Contact page
```
Comment (hawkrives) [inline source/views/contacts/contact-list.js]: Remove the word iOS?

**Review by hawkrives — CHANGES_REQUESTED** [review]
Overall, nice work! Really nice!

**Comment by drewvolz** [comment]
Changes addressed. Now to wait on the build again.

**Review by hawkrives — APPROVED** [review]

**Pull request merged**